presto-main/src/main/java/com/facebook/presto/SystemSessionProperties.java

```diff
@@ -268,6 +268,7 @@ public final class SystemSessionProperties
     public static final String SEGMENTED_AGGREGATION_ENABLED = "segmented_aggregation_enabled";
     public static final String USE_HISTORY_BASED_PLAN_STATISTICS = "use_history_based_plan_statistics";
     public static final String TRACK_HISTORY_BASED_PLAN_STATISTICS = "track_history_based_plan_statistics";
+    public static final String TRACK_HISTORY_STATS_FROM_FAILED_QUERY = "track_history_stats_from_failed_query";
```

Comment:
```suggestion
    public static final String TRACK_HISTORY_STATS_FROM_FAILED_QUERies = "track_history_stats_from_failed_queries";
```

and update method names accordingly